test/Arrays/broadcasting.jl

```diff
@@ -16,8 +16,8 @@ const mpicomm = MPI.COMM_WORLD
         QA = MPIStateArray{Float32}(mpicomm, ArrayType, localsize...)
         QB = similar(QA)
 
-        QA .= A
-        QB .= B
+        copyto!(QA, A)
```

Comment:
@lcw @jkozdon I think it is cleaner to make the data-movement explicit instead of having it be specialised on a particular form of broadcast. Heterogenous broadcasts are not supported as so I could see a user get confused about this one special case.

Reply:
I thought this was originally added for the ODE solver stuff, but maybe @mwarusz knows (he added all the original broadcasting stuff)